deploy/helm/sumologic/values.yaml

```diff
@@ -163,9 +163,6 @@ sumologic:
   ## This is experimental feature and may be unavailable for your account
   traces:
     enabled: false
-    ## Use filter stdout plugin for tracing data
-    ## Warning: It will produce additional fluentd logs (one per trace)
-    fluentd_stdout: false
```

Comment:
Why this? Is it a breaking change?

Reply:
I believe this is a deprecated piece from when `fluentd` was taking care of traces. I didn't find any reference to it anywhere in the repo anymore

Reply:
correct 👍